javascript-injectors/README.md

```diff
@@ -104,4 +103,4 @@ For more details see the `window.ScrollBy()` [documentation](https://developer.m
 <img src="../images/engadget-logo.png" alt="Engadget logo" width="200"/>
 
 * Download [engadget-cookies-consent-closing.js](https://github.com/Screenly/playground/tree/master/javascript-injectors/engadget-cookies-consent-closing.js)
-* PATCH the asset.
+* PATCH the .
```

Comment:
Accidental delete?

Reply:
Ah, that's from another diff. Remember that we went away from the `asset` word and use `content` now.
I will undo this change.

Reply:
We still call it assets in the API, so i think we should call it assets here too.